documentation/docs/getting_started/java.mdx

```diff
@@ -1,5 +1,6 @@
 ---
```

Comment:
Maybe we can import the README here with the !!raw_loader as well.
Theyre basically the same except that the import wont work.

Could get around that by making the readme start at the security segment (Which is empty?)

Reply:
Hmm not so easy...
Is there a way to only import a part? for example only import from line X to line Y?